test/e2e/tests/local/test_realtime.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Cortex Labs, Inc.
```

Comment:
Since we're removing local, it might make sense to remove this from the PR, or we could delete it after merging, whichever you prefer

Reply:
makes sense, removed